vite.config.js

```diff
@@ -0,0 +1,8 @@
+import { defineConfig } from 'vite'
+
+import pugPlugin from './scripts/pugPlugin.js'
+
+export default defineConfig({
+  plugins: [pugPlugin()],
```

Comment:
Please add additional options from [this commit](https://github.com/logux/logux.io/commit/476cba2d506a1554f32a3b025ee39910128581d1)